diesel_migrations/migrations_internals/src/lib.rs

```diff
@@ -174,11 +174,10 @@ where
 ///
 /// See the [module level documentation](index.html) for information on how migrations should be
 /// structured, and where Diesel will look for them by default.
-pub fn any_pending_migrations<Conn>(conn: &Conn) -> Result<bool, RunMigrationsError>
+pub fn any_pending_migrations<Conn>(conn: &Conn, migrations_dir: &Path) -> Result<bool, RunMigrationsError>
```

Comment:
That's a breaking change. I would rather like to not doing that. Instead add a new method named `any_pending_migrations_in_directory` containing the relevant part of this method. Then the existing method should just internally call the new method with the infered migration directory.